en/sync-themes-plugin.md

```diff
@@ -108,7 +108,7 @@ You have a Sketch project that you feel is robust enough to share with others. Y
 - choose to highlight hotspots and interactions
 - allow comments on the design
 
-Once you click the Publish button, the project is uploaded to the Indigo.Design Cloud and you will get a link to it, which you may share for feedback or use to [generate code](codegen/vscode-plugin.md).
+Once you click the Publish button, the project is uploaded to the Indigo.Design Cloud and you will get a link to it, which you may share for feedback or use to generate code.
```

Comment:
I don't think we can generate code from the prototype that gets uploaded to Indigo.Design Cloud in any other way besides the vs code plugin. I see no such option from the prototype UI, nor in the app builder itself.
So probably the 'generate code' part can be removed altogether.